packages/libs/react-ui/src/components/TrackerCard/TrackerCard.tsx

```diff
@@ -0,0 +1,120 @@
+import { MaskedValue, ProductIcon } from '..';
+
+import {
+  CardContainer,
+  ContentContainer,
+  DataContainer,
+  displayVariant,
+  gapValueLabelVariant,
+  gridVariant,
+  LabelTitle,
+  LabelValue,
+  LabelValueContainer,
+  layoutVariant,
+  TrackerWarningContainer,
+  warningVariant,
+} from './TrackerCard.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface ITrackerCardProps {
+  labelValue: ILabelValue[];
+  helperText?: string;
+  helperTextType?: 'mild' | 'severe';
+  icon?: (typeof ProductIcon)[keyof typeof ProductIcon];
+  variant: keyof typeof layoutVariant;
+}
+
+export interface ILabelValue {
+  label: string;
+  value?: string;
+  isAccount?: boolean;
+  defaultVisible?: boolean;
+  startUnmasked?: number;
+  endUnmasked?: number;
+}
+
+export const TrackerCard: FC<ITrackerCardProps> = ({
+  labelValue,
+  icon,
+  helperText,
+  helperTextType = 'mild',
+  variant,
+}): JSX.Element => {
+  const classCardContainer = classNames(
+    CardContainer,
+    icon ? layoutVariant[variant] : null,
+    gridVariant[variant],
+  );
+
+  const classLabelValue = classNames(
+    LabelValueContainer,
+    displayVariant[variant],
+    layoutVariant[variant],
+    gapValueLabelVariant[variant],
+  );
+
+  const classWarningContainer = classNames(
+    TrackerWarningContainer,
+    warningVariant[helperTextType],
+  );
+
+  const Icon = icon;
+
+  return (
+    <div className={classCardContainer} data-testid="kda-tracker-card">
+      {Icon ? (
+        <>
+          <Icon data-testid="kda-icon" />
+        </>
+      ) : (
+        <div />
```

Comment:
Do we want to render an empty div here, or could we just use `null` here?

Reply:
I am rendering an empty `div` to ensure that the remaining content gets allocated to the second row/column (depending on the variant) and therefore the correct styling is applied to it.
With empty div and no icon.
![image](https://github.com/kadena-community/kadena.js/assets/112877112/6ab76a6d-e7f4-4275-ad1a-b6f5f9ffd5be)

With null and empty icon
![image](https://github.com/kadena-community/kadena.js/assets/112877112/4075c985-bdd1-4d96-ac2c-3be9bc8422af)

Reply:
You were clear, thanks for clarifying 👍